Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM owncloudops/nginx
+
+RUN rm -f /var/lib/nginx/html/*
+
+ADD docker/vhost.conf /etc/nginx/vhost.conf
+ADD dist /var/lib/nginx/html
```

Comment:
should dockerfile be able to build the extension with `docker build` command? That should prevent the extra build extension command before running `docker build`